src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -1884,6 +1884,19 @@ public function isClean($attributes = null)
         return ! $this->isDirty(...func_get_args());
     }
 
+    /**
+     * Discard attribute(s) changes.
+     *
+     * @return $this
+     */
+    public function discardChanges()
```

Comment:
Maybe `reset`?
```suggestion
    public function reset()
```

Reply:
or `rollback()` ?

Reply:
Tbh `clean` makes most sense, since we have getDirty and isClean as I mentioned [here](https://github.com/laravel/framework/pull/43772#pullrequestreview-1078976382)